internal/v1/handler.go

```diff
@@ -494,6 +494,92 @@ func (h *Handlers) GetComposeMetadata(ctx echo.Context, composeId uuid.UUID) err
 	return ctx.JSON(http.StatusOK, status)
 }
 
+func (h *Handlers) GetComposeSBOMs(ctx echo.Context, composeId uuid.UUID, params GetComposeSBOMsParams) error {
+	err := h.canUserAccessComposeId(ctx, composeId)
+	if err != nil {
+		return err
+	}
+
+	limit := 100
+	if params.Limit != nil && *params.Limit > 0 {
+		limit = *params.Limit
+	}
+
+	offset := 0
+	if params.Offset != nil {
+		offset = *params.Offset
+	}
+
+	resp, err := h.server.cClient.ComposeSBOMs(ctx.Request().Context(), composeId)
+	if err != nil {
+		return err
+	}
+	defer closeBody(ctx, resp.Body)
+
+	if resp.StatusCode == http.StatusNotFound {
+		body, err := io.ReadAll(resp.Body)
+		if err != nil {
+			return err
+		}
+		return echo.NewHTTPError(http.StatusNotFound, string(body))
+	} else if resp.StatusCode != http.StatusOK {
+		httpError := echo.NewHTTPError(http.StatusInternalServerError, "Failed querying compose SBOMs")
+		body, err := io.ReadAll(resp.Body)
+		if err != nil {
+			ctx.Logger().Errorf("unable to parse composer's compose SBOMs response: %v", err)
```

Comment:
It would be slightly more accurate for this to say "unable to *read* composer's..."